webpack.config.js

```diff
@@ -78,10 +83,36 @@ if (process.env.NODE_ENV === "production") {
   plugins.push(new webpack.EnvironmentPlugin(["REMOTE_DEV_SERVER_PORT"]));
 }
 
+// telemetry configuration
+let telemetryAppId = "org-mozilla-bergamot-test";
```

Comment:
We probably only want to enable -test app ids when running tests. I suggested however in https://bugzilla.mozilla.org/show_bug.cgi?id=1696303#c15 that we stop using this. We'll find other ways to make sure it works and can be tested properly.

Reply:
at this point this is a way to go, we don't have debug tags support now